ext/opcache/ZendAccelerator.c

```diff
@@ -3559,75 +3772,77 @@ static void get_unlinked_dependency(zend_class_entry *ce, const char **kind, con
 
 static bool preload_try_resolve_constants(zend_class_entry *ce)
 {
-	bool ok, changed;
+	bool ok, changed, was_changed = 0;
 	zend_class_constant *c;
 	zval *val;
 
 	EG(exception) = (void*)(uintptr_t)-1; /* prevent error reporting */
-	CG(in_compilation) = 1; /* prevent autoloading */
```

Comment:
Why is this removed? Shouldn't we still suppress autoloading for `opcache_compile_file` based preloading?

Reply:
This assignment is moved into preload_link(). preload_try_resolve_constants() is used from two different places now.